new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSystemResOverrides.java

```diff
@@ -0,0 +1,514 @@
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDomainAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createDomainJob;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createPV;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createPVC;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getPodCreationTime;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.installAndVerifyOperator;
+import static oracle.weblogic.kubernetes.utils.DeployUtil.deployUsingWlst;
+import static oracle.weblogic.kubernetes.utils.TestUtils.getNextFreePort;
+import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests related to Situational Configuration overrides for system resources.
+ */
+@DisplayName("Verify the JMS and WLDF system resources are overridden with values from override files")
+@IntegrationTest
+public class ItSystemResOverrides {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private static String image = WLS_BASE_IMAGE_NAME + ":" + WLS_BASE_IMAGE_TAG;
+  private static boolean isUseSecret = true;
+
+  final String domainUid = "mydomain";
+  final String clusterName = "mycluster";
+  final String adminServerName = "admin-server";
+  final String adminServerPodName = domainUid + "-" + adminServerName;
+  final String managedServerNameBase = "ms-";
+  final int managedServerPort = 8001;
+  int t3ChannelPort;
+  final String pvName = domainUid + "-pv"; // name of the persistent volume
+  final String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+  final String wlSecretName = "weblogic-credentials";
+  final String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+  int replicaCount = 2;
+
+  static Path sitconfigAppPath;
+  String overridecm = "configoverride-cm";
+  LinkedHashMap<String, DateTime> podTimestamps;
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+  private static LoggingFacade logger = null;
+
+  /**
+   * Assigns unique namespaces for operator and domains.
+   * Pulls WebLogic image if running tests in Kind cluster.
+   * Installs operator.
+   * Creates and starts WebLogic domain containing 2 instances in dynamic cluster.
```

Comment:
Does it mean two dynamic clusters with two managed server each ?
Specify which domain model is being used.

Reply:
changed the wording, it is 1 domain, 1 cluster and 2 ms in it.